src/applicationInsightsClient.ts

```diff
@@ -25,10 +29,15 @@ export class ApplicationInsightsClient {
                 "Connection String not found, please provide it before starting Application Insights SDK."
             );
         }
-        if (!this._config.getDisableStatsbeat()) {
-            this._statsbeat = new Statsbeat(this._config);
-            this._statsbeat.enable(true);
-        }
+        this._getStatsbeatInstrumentations();
+        this._getStatsbeatFeatures();
+        process.env.AZURE_MONITOR_STATSBEAT_FEATURES = JSON.stringify({
```

Comment:
Use constants for this env variable and try/catch all JSON.stringify calls

Reply:
I moved this logic to the `statsbeat.ts` file and added the try/catch logic to handle any parsing errors from JSON.stringify. I included a const to give the value of the environment variable more visibility in the Statsbeat file, were you looking for me to change where the `process.env` call is made to set the environment variable? Just want to make sure I'm conforming to best practices here.